python/cuml/test/test_linear_model.py

```diff
@@ -21,80 +21,87 @@
+@pytest.mark.parametrize('n_info', [pytest.param(2, marks=pytest.mark.unit),
+                                    pytest.param(500,
+                                                 marks=pytest.mark.stress),
+                                    pytest.param(50,
+                                                 marks=pytest.mark.quality)])
+def test_unit(datatype, X_type, y_type, algorithm, nrows, ncols, n_info):
+    n_info = 2
+    train_rows = np.int32(nrows*0.8)
+    X, y = make_regression(n_samples=(nrows), n_features=ncols,
+                           n_informative=n_info, random_state=0)
+    X_test = np.asarray(X[train_rows:, 0:]).astype(datatype)
+    X_train = np.asarray(X[0:train_rows, :]).astype(datatype)
+    y_train = np.asarray(y[0:train_rows, ]).astype(datatype)
+
+    # sklearn linear and ridge regression model initialization and fit
     skols = skLinearRegression(fit_intercept=True,
                                normalize=False)
-    skols.fit(X, y)
+    skols.fit(X_train, y_train)
+    skridge = skRidge(fit_intercept=False,
+                      normalize=False)
+    skridge.fit(X_train, y_train)
 
+    # Initialization of cuML's linear and ridge regression models
     cuols = cuLinearRegression(fit_intercept=True,
                                normalize=False,
                                algorithm=algorithm)
 
-    if X_type == 'dataframe':
-        gdf = cudf.DataFrame()
-        gdf['0'] = np.asarray([2, 6, 2, 2], dtype=datatype)
-        gdf['1'] = np.asarray([5, 9, 2, 3], dtype=datatype)
-        cuols.fit(gdf, y)
-
-    elif X_type == 'ndarray':
-        cuols.fit(X, y)
-
-    sk_predict = skols.predict(pred_data)
-    cu_predict = cuols.predict(pred_data).to_array()
-
-    print(sk_predict)
-    print(cu_predict)
-
-    # print(skols.coef_)
-    print(cuols.gdf_datatype)
-    print(y.dtype)
-
-    assert array_equal(sk_predict, cu_predict, 1e-3, with_sign=True)
-
-
-@pytest.mark.parametrize('datatype', [np.float32, np.float64])
-@pytest.mark.parametrize('X_type', ['dataframe', 'ndarray'])
-@pytest.mark.parametrize('y_type', ['series', 'ndarray'])
-@pytest.mark.parametrize('algorithm', ['eig', 'svd'])
-def test_ridge(datatype, X_type, y_type, algorithm):
-
-    X = np.array([[2.0, 5.0], [6.0, 9.0], [2.0, 2.0], [2.0, 3.0]],
-                 dtype=datatype)
-    y = np.dot(X, np.array([5.0, 10.0]).astype(datatype))
-
-    pred_data = np.array([[3.0, 5.0], [2.0, 5.0]]).astype(datatype)
-
-    skridge = skRidge(fit_intercept=False,
-                      normalize=False)
-    skridge.fit(X, y)
-
     curidge = cuRidge(fit_intercept=False,
                       normalize=False,
                       solver=algorithm)
 
     if X_type == 'dataframe':
-        gdf = cudf.DataFrame()
-        gdf['0'] = np.asarray([2, 6, 2, 2], dtype=datatype)
-        gdf['1'] = np.asarray([5, 9, 2, 3], dtype=datatype)
-        curidge.fit(gdf, y)
+        y_train = pd.DataFrame({'fea0': y_train[0:, ]})
```

Comment:
Super-picky/borderline-stupid review comment: I would name this `target0` or something like that... feels weird to have both the output and the predictor called fea0

Reply:
changed the column name to `labels`